torchvision/transforms/functional_tensor.py

```diff
@@ -880,7 +880,7 @@ def _scale_channel(img_chan: Tensor) -> Tensor:
     if img_chan.is_cuda:
         hist = torch.histc(img_chan.to(torch.float32), bins=256, min=0, max=255)
     else:
-        hist = torch.bincount(img_chan.view(-1), minlength=256)
```

Comment:
Without this we get:
```
    hist = torch.bincount(img_chan.view(-1), minlength=256)
RuntimeError: view size is not compatible with input tensor's size and stride (at least one dimension spans across two contiguous subspaces). Use .reshape(...) instead.
```

Reply:
Is this still relevant after #6726?

Reply:
No idea. I'll need to check.

Reply:
yeap same issue on the new implemantation:
```
  File "./torchvision/prototype/transforms/functional/_color.py", line 237, in equalize_video
    return equalize_image_tensor(video)
  File "./torchvision/prototype/transforms/functional/_color.py", line 230, in equalize_image_tensor
    return _equalize_image_tensor_vec(image.view(-1, height, width)).view(image.shape)
  File "./torchvision/prototype/features/_feature.py", line 94, in __torch_function__
    output = func(*args, **kwargs or dict())
RuntimeError: view size is not compatible with input tensor's size and stride (at least one dimension spans across two contiguous subspaces). Use .reshape(...) instead.
```